File: Java/src/pl/eco/zwierzeta/Owca.java
package pl.eco.zwierzeta;

import pl.eco.Swiat;
import pl.eco.Zwierze;

public class Owca extends Zwierze {
    public Owca(int x, int y, Swiat moj) {
        sila = 4;
        inicjatywa = 4;
        nazwa = "owca";
        symbol = 'O';
        wiek = 0;
        poRozmnozeniu=0;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
    public Owca(int x, int y, Swiat moj, int Sila, int Wiek, int poRozmnazaniu) {
        this.sila = sila;
        this.wiek = Wiek;
        this.poRozmnozeniu=poRozmnazaniu;
        inicjatywa = 4;
        nazwa = "owca";
        symbol = 'O';
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
}

File: Python/Zwierzeta/Owca.py
from Zwierze import Zwierze


class Owca(Zwierze):
    def __init__(self, x, y, moj):
        self.sila = 4
        self.inicjatywa = 4
        self.symbol = 'O'
        self.nazwa = "owca"
        self.wiek = 0
        self.poRozmnozeniu = 0
        self.x = x
        self.y = y
        self.moj = moj
        self.czyZyje = True

    def generator(self, x, y):
        return Owca(x, y, self.moj)
